helm/cluster-autoscaler-app/templates/rbac.yaml

```
@@ -81,7 +81,7 @@ rules:
 - apiGroups:
   - storage.k8s.io
   resources:
-  - storageclasses
+  - storageclasses, csinodes
```

This is not correct.
```suggestion
  - storageclasses
  - csinodes
```

---

thanks :)